community/kernel/src/test/java/org/neo4j/test/subprocess/SubProcess.java

```diff
@@ -214,7 +197,7 @@ private String classPath( String parentClasspath )
 
     private static Process start( String... args )
     {
-        ProcessBuilder builder = new ProcessBuilder( args );
+        ProcessBuilder builder = new ProcessBuilder( args ).inheritIO();
```

Comment:
Having had time to look at it, the reason is right there in the commit message: 95fe9d6f84a23f828bd547554732da122db5a212

> We have tests that rely on being able to communicate with sub-processes through stdin and stdout. Simply letting the subprocess inheriting the IO streams from the parent process does not allow such communication.

I still don't remember exactly which test cases it was that actually used the ability to communicate with the subprocess through stdio, but I think it is a very bad idea to remove the capability to do so.

How about instead of forcing all subprocesses to inherit IO, which is what this PR does, you add the ability to configure a particular SubProcess to inherit IO, so that tests that want this behaviour can opt in to it?